### ISSUE TITLE
[Feature Request] Allow for multiple job-locations

### ISSUE THREAD

**Issue by dannyqwertz** [itx-informationssysteme/jobapplications#21]
It would be of use to allow backend-selection for multiple job-locations.

Google explicitly allows this for JSON-LD in the following way:

```
Multiple physical locations

If the job has multiple locations, add multiple jobLocation properties in an array.
Google will choose the best location to display based on the job seeker's query.
```

This could be selectable in the backend just like the employment_type field:

![grafik](https://user-images.githubusercontent.com/3381961/99966850-3b1d0000-2d97-11eb-84b4-3578aba57be4.png)

**Comment by itx-benjamin-jasper** [comment]
This is actually right and therefore a good feature. Will have to check if this is breaking. If it is, we will include this in the next breaking release.

**Comment by itx-stefanie-doell** [comment]
the new functionality is now implemented in Version 2.0.0
